solr/core/src/java/org/apache/solr/handler/admin/ClusterStatus.java

```diff
@@ -45,7 +45,11 @@
 public class ClusterStatus {
   private final ZkStateReader zkStateReader;
   private final ZkNodeProps message;
-  private final String collection; // maybe null
+  private final String collectionParam; // maybe null
+  private final boolean liveNodesParam;
+  private final boolean clusterPropertiesParam;
+  private final boolean rolesParam;
+  private final boolean includeAll;
```

Comment:
Instead see `org.apache.solr.handler.admin.ColStatus#getColStatus` for better naming, like "withSegments" etc.

Reply:
Addressed in latest commit